src/cpp/server/server_cc.cc

```diff
@@ -382,11 +382,13 @@ Server::Server(
   global_callbacks_ = g_callbacks;
   global_callbacks_->UpdateArguments(args);
 
-  for (auto it = sync_server_cqs_->begin(); it != sync_server_cqs_->end();
-       it++) {
-    sync_req_mgrs_.emplace_back(new SyncRequestThreadManager(
-        this, (*it).get(), global_callbacks_, min_pollers, max_pollers,
-        sync_cq_timeout_msec));
+  if (sync_server_cqs_ != nullptr) {
+    for (auto it = sync_server_cqs_->begin(); it != sync_server_cqs_->end();
+         it++) {
```

Comment:
So we used to use the `begin`/`end` format all the time because of supporting pre-C++11 compilers, but we don't anymore, so can you change this to a range? I'm asking everyone to do this opportunistically any time they make a change in the C++ wrapping that affects one of these.

Reply:
done